Bottom: a3b3296bcfe508b80c37e1f457f2004af21ec377
Top:    028fb5c428aa28ab0b9cff275b63584febb2c170
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-04-30 23:30:56 +0100

Refresh of alternative-more-thorough-step

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 14c2970..93a7ceb 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -191,17 +191,7 @@ LIBGTK_GDK_SRCS= \
 LIBGTK_GL_SRCS= \
 	hid/gtk/gtkhid-gl.c \
 	hid/gtk/gui-trackball.c \
-	hid/gtk/gui-trackball.h \
-	hid/gtk/quad.c \
-	hid/gtk/quad.h \
-	hid/gtk/vertex3d.c \
-	hid/gtk/vertex3d.h \
-	hid/gtk/face3d.c \
-	hid/gtk/face3d.h \
-	hid/gtk/edge3d.c \
-	hid/gtk/edge3d.h \
-	hid/gtk/object3d.c \
-	hid/gtk/object3d.h
+	hid/gtk/gui-trackball.h
 
 GL_SRCS= \
 	hid/common/hidgl.c \
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 2332d07..b2539ca 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -15,8 +15,14 @@
 #include "gui-pinout-preview.h"
 #include "pcb-printf.h"
 
-#include "quad.h"
-#include "object3d.h"
+#include "hid/common/step_id.h"
+#include "hid/common/quad.h"
+#include "hid/common/vertex3d.h"
+#include "hid/common/contour3d.h"
+#include "hid/common/appearance.h"
+#include "hid/common/face3d.h"
+#include "hid/common/edge3d.h"
+#include "hid/common/object3d.h"
 
 #ifndef WIN32
 /* The Linux OpenGL ABI 1.0 spec requires that we define
